### ISSUE TITLE
docker-compose up doesn't work

### ISSUE THREAD

**Issue by valeriocos** [crossminer/scava-deployment#87]
As commented at https://github.com/crossminer/scava-deployment/issues/85#issuecomment-520717806, when doing `docker-compose up` the process gets blocked. This situation makes that nothing (workers, projects, metrics, etc.) is visible on the cockpit UI except for http://localhost:5601/#/home, and metrics are not continously imported to Elasticsearch/Kibana (this means that the metrics will be imported just once, and the loop won't start again). In my specific case, I cannot advance on my scenarios for validation (cc @davidediruscio).

I've inspected the logs of each container and it seems there is something wrong within `scava-deployment_kb-db`.

```
...
2019-08-14T14:01:16.241+0000	CROSSMINER.mavenLinkAll  384MB
2019-08-14T14:01:16.241+0000	CROSSMINER.mavenLibrary  272MB
2019-08-14T14:01:16.241+0000	    CROSSMINER.artifact  175MB
2019-08-14T14:01:16.241+0000	
2019-08-14T14:01:19.240+0000	CROSSMINER.mavenLinkAll  384MB
2019-08-14T14:01:19.240+0000	CROSSMINER.mavenLibrary  272MB
2019-08-14T14:01:19.240+0000	    CROSSMINER.artifact  175MB
2019-08-14T14:01:19.240+0000	
2019-08-14T14:01:19.775+0000	finished restoring CROSSMINER.relation (1420355 documents)
2019-08-14T14:01:22.242+0000	CROSSMINER.mavenLinkAll  400MB
2019-08-14T14:01:22.242+0000	CROSSMINER.mavenLibrary  291MB
2019-08-14T14:01:22.242+0000	    CROSSMINER.artifact  175MB
2019-08-14T14:01:22.242+0000	
2019-08-14T14:01:22.905+0000	reading metadata for CROSSMINER.pattern from archive 'CROSSMINER.gz'
2019-08-14T14:01:22.912+0000	Failed: CROSSMINER.pattern: error creating collection CROSSMINER.pattern: error running create command: a collection 'CROSSMINER.pattern' already exists
```

@MarcioMateus , @md2manoppello , @creat89 are you seeing the same problem? Do you have some workarounds to suggest?

EDITED: 
This PR fixes an error https://github.com/crossminer/scava-deployment/pull/88 seen when checking the logs of each container.

**Comment by valeriocos** [comment]
I'm leaving the url http://localhost:5601/#/project open, maybe the UI takes time to get updated.

**Comment by valeriocos** [comment]
I resumed today the analysis of the bug. The dashb-importer was executed just once and the worker is still alive. However nothing was visible on the cockpit UI except for home.

The last trace in the docker-compose log was a message generated by Prosoul. In order to check that Prosoul wasn't causing any error, I stopped the docker-compose, relaunch it and stop the Prosoul container.

The last traces in the docker-compose log are now messages of `kb-db` (see below). The cockpit UI doesn't show anything, thus it isn't possible analyse any project.

```
kb-db_1           | 2019-08-15T08:25:50.060+0000	finished restoring CROSSMINER.mavenLibrary (2407335 documents)
...
kb-db_1           | 2019-08-15T08:26:21.545+0000	CROSSMINER.mavenLinkAll  1.44GB
kb-db_1           | 2019-08-15T08:26:24.545+0000	CROSSMINER.mavenLinkAll  1.53GB
kb-db_1           | 2019-08-15T08:26:27.545+0000	CROSSMINER.mavenLinkAll  1.61GB
kb-db_1           | 2019-08-15T08:26:30.545+0000	CROSSMINER.mavenLinkAll  1.69GB
kb-db_1           | 2019-08-15T08:26:33.545+0000	CROSSMINER.mavenLinkAll  1.77GB
kb-db_1           | 2019-08-15T08:26:36.545+0000	CROSSMINER.mavenLinkAll  1.86GB
kb-db_1           | 2019-08-15T08:26:37.118+0000	CROSSMINER.mavenLinkAll  1.87GB
kb-db_1           | 2019-08-15T08:26:37.118+0000	restoring indexes for collection CROSSMINER.mavenLinkAll from metadata
```

**Comment by valeriocos** [comment]
I tried also with the docker-compse in the ow2 branch, and also there the metrics are not continously imported to Elasticsearch/Kibana, and the cockpit UI shows nothing.
I commented the prosoul and dash-importer sections in the docker-compose and now I'm able to see projects in the UI.

The next step is to check what has changed in the docker-compose (of dev/ow2 branch) that is preventing the dash-importer to continously import data to elasticsearch.
